devstack/lib/nfp

```diff
@@ -123,10 +123,6 @@ function create_ep_and_nsp {
  --shared True\
  default
 
-    gbp ep-create\
- --external-segments default\
- ext_connect
```

Comment:
What about non-VPN services ?
Are you saying that, the ep is created implicitly in any chain during ptg create. Or, in VPN explicitly we are creating ?